codeforamerica/vita-min
# Add team assignments module

## Link to pivotal/JIRA issue
- e.g. [link](https://codeforamerica.atlassian.net/browse/GYR1-501?atlOrigin=eyJpIjoiMDc4NjNjNDAyNWZkNDc3ZjkyNTUzMDU4YjVjZTMzYjYiLCJwIjoiaiJ9)
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!
## What was done?
- Added a "team assignments" module to the hub dashboard
- This module is used by organization leads and site coordinators to be able to view other users and how many assigned tax returns they have
## How to test?
  - test in heroku, login as an org lead or a site coordinator ("cucumber@example.com" or "skywalker@example.com" logins)
  - this view will not show up for coalition leads or other users
- Risk Assessment
  - check that only displays accessible users 
## Screenshots (for visual changes)
- Before
![Screenshot 2024-07-31 at 12 50 26 PM](https://github.com/user-attachments/assets/f5d49c00-688e-4cc4-98b1-dc290f4466d8)

- After
<img width="427" alt="Screenshot 2024-07-30 at 8 35 28 PM" src="https://github.com/user-attachments/assets/d8e43e67-a72d-49f9-864a-cdbbd5f98102">
